hackle-android-sdk/src/main/java/io/hackle/android/internal/user/UserTargetEventFetcher.kt

```diff
@@ -73,5 +76,22 @@ internal fun UserTargetRequestDto.encodeBase64Url(): String {
 }
 
 internal class UserTargetResponseDto(
-    val events: List<TargetEvent>
+    val events: List<TargetEventDto>
+)
+
+internal class TargetEventDto(
+    val eventKey: String,
+    val stats: List<StatDto>,
+    val property: PropertyDto?
+)
+
+internal class StatDto(
+    val date: Long,
+    val count: Int
+)
+
+internal class PropertyDto(
+    val key: String,
+    val type: Target.Key.Type,
```

Comment:
```suggestion
    val type: String,
```
sdk에서 지원하지 않는 타입이 추가 되었을때 커버를 위해서 DTO에는 string으로 하고 실제 모델로 변환할때 enum type 으로 변경합니다.